### PR TITLE
simplyemail: add necesary dependences

### LINKED ISSUE #2041: [BUG] SimplyEmail install
It seems that multiple dependencies are missing for SimplyEmail package.
We need to add these one : python2-lxml python2-mechanize python2-beautifulsoup3 python2-fake-useragent python2-magic

### PR DESCRIPTION
**Pull request by Edu4rdSHL**
It fix  #2041

### CODE REVIEW

**Comment by FFY00** [comment]
Can you please make both dependency lines of the same size? Thanks

File: packages/simplyemail/PKGBUILD [hunk 1]
```diff
@@ -9,9 +9,8 @@ groups=('blackarch' 'blackarch-recon')
 url='https://github.com/killswitch-GUI/SimplyEmail'
 license=('GPL2')
 arch=('any')
-depends=('python2' 'python2-requests' 'python2-pdfminer' 'python2-docx2txt'
-         'python2-beautifulsoup4' 'python2-xlsx2csv' 'antiword' 'odt2txt'
-         'wget')
+depends=('python2' 'python2-requests' 'python2-pdfminer' 'python2-docx2txt' 'python2-beautifulsoup4' 'python2-xlsx2csv' 'python2-lxml' \
```
Comment (FFY00) [inline packages/simplyemail/PKGBUILD]: No need `\` at end of line.

File: packages/simplyemail/PKGBUILD [hunk 2]
```diff
@@ -35,10 +34,10 @@ package() {
...
   cat > "$pkgdir/usr/bin/simplyemail" << EOF
-#!/bin/sh
-cd /usr/share/simplyemail
-exec python2 SimplyEmail.py "\$@"
-EOF
-
+  #!/bin/sh
+  cd /usr/share/simplyemail
+  exec python2 SimplyEmail.py "\$@"
+  EOF
```
Comment (FFY00) [inline packages/simplyemail/PKGBUILD]: No need.

File: packages/simplyemail/PKGBUILD [hunk 3]
```diff
@@ -35,10 +34,10 @@ package() {
...
+
```
Comment (FFY00) [inline packages/simplyemail/PKGBUILD]: Whitespace.

**Comment by y0no** [comment]
👍

**Comment by y0no** [comment]
You need to increment pkgrel to diffuse the upgrade correctly on old install too.

**Comment by noptrix** [comment]
i've updated the pkgbuild file and releaed new revision of simplyemail. please check.

**Pull request merged**